x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/WatcherLifeCycleServiceTests.java

```diff
@@ -69,8 +73,12 @@
     private WatcherService watcherService;
     private WatcherLifeCycleService lifeCycleService;
 
+    @Captor
+    private ArgumentCaptor<Consumer<Void>> voidConsumerCaptor;
+
     @Before
     public void prepareServices() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
this is knowledge lacking on my side: Why do we need this? Because of the annotation above? Couldnt that just be assigned to an argument captor in order to not need the annotation and this method?

Reply:
this was a workaround to an issue with Mockito and Generics (simply to avoid warnings). 

However, this has been removed now that `Consumer<Void>` has been replaced by `Runnable`